app/views/tweets/index.erb

```diff
@@ -0,0 +1,6 @@
+Welcome, <%= @user.username %>
+<% Tweet.all.each do |tweet| %>
```

Comment:
With a MVC design patter you want to avoid Database calls in the view. The controller should interact with the model for this query instead 

```ruby 
def index 
  @tweets = Tweet.all 
end 
```